### PR TITLE
Add missing description to barclamp YAMLs

### PR DESCRIPTION
**Pull request by rsalevsky**

### CODE REVIEW

**Comment by toabctl** [comment]
+1

File: nfs_client.yml [hunk 1]
```diff
@@ -17,6 +17,7 @@
 barclamp:
   name: 'nfs_client'
   display: 'NFS Client'
+  description: 'NFS Mount Configuration'
```
Comment (vuntz) [inline nfs_client.yml]: I currently see "Setup of NFS mounts" in the crowbar webui, which I slightly prefer to the above.

File: ntp.yml [hunk 2]
```diff
@@ -18,6 +18,7 @@
 barclamp:
   name: 'ntp'
   display: 'NTP'
+  description: 'Common NTP service for the cluster'
```
Comment (vuntz) [inline ntp.yml]: Can we take this as an opportunity to get a better string? :-) Either "NTP Management" like for many other barclamps or something else.

**Comment by tboerger** [comment]
:+1: Can you update my registry pull request as well? :)

**Comment by rsalevsky** [comment]
@tboerger Will do this when we work an the card.

**Pull request merged**